tests/integration/pybaseball/test_statcast_fielding.py

```diff
@@ -39,7 +39,7 @@ def test_statcast_outfield_catch_proba() -> None:
 	assert not result.empty
 
 	assert len(result.columns) == 19
-	assert len(result) == 228
+	assert len(result) == 224
```

Comment:
Relax this assert.
After all, what is 224 if is not just a positive number 😆 / 😭?